code/modules/antagonists/heretic/knowledge/cosmic_lore.dm

```diff
@@ -191,7 +191,7 @@
 	second_target = null
 	third_target = null
 	if(combo_counter > 3)
-		source.RemoveElement(/datum/element/effect_trail/cosmic_trail)
+		source.RemoveElement(/datum/element/effect_trail)
```

Comment:
You also have to provide the same argument when the element is removed, or it will not be removed properly.